Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -140,6 +143,14 @@
   public long maxMillisToWaitForServerResponse = 0L;
   @NotNull private final InteractiveErrorReporter myErrorReporter = new InteractiveErrorReporter();
 
+  @NotNull private final Object myDebugLogLock = new Object();
```

Comment:
That caused IntelliJ to signal the field was accessed in both sync and async contexts.